pr_agent/servers/bitbucket_app.py

```diff
@@ -219,7 +282,19 @@ async def inner():
                         if get_identity_provider().verify_eligibility("bitbucket",
                                                         sender_id, pr_url) is not Eligibility.NOT_ELIGIBLE:
                             if get_settings().get("bitbucket_app.pr_commands"):
-                                await _perform_commands_bitbucket("pr_commands", PRAgent(), pr_url, log_context, data)
+                                await _perform_commands_bitbucket("pr_commands", agent, pr_url, log_context, data)
+            elif event == "pullrequest:updated": # PR updated, might be from a push (we will validate this later)
+                pr_url = data["data"]["pullrequest"]["links"]["html"]["href"]
+                log_context["api_url"] = pr_url
+                log_context["event"] = "pull_request"
+                if pr_url:
+                    with get_logger().contextualize(**log_context):
+                        apply_repo_settings(pr_url)
```

Comment:
Please validate that this is necessary here; we have it within the `_perform_commands_bitbucket` method.